File: programs/fib/fib.go
package main

import "fmt"

func fib(n int64) int64 {
    if n < 2 {
        return n
    }
    return fib(n-1) + fib(n-2)
}

func main() {
    fmt.Printf("fib(35) = %d\n", fib(35))
}

File: programs/fib/fib.cpp
#include <cstdio>

static long fib(long n) {
    if (n < 2) return n;
    return fib(n - 1) + fib(n - 2);
}

int main() {
    printf("fib(35) = %ld\n", fib(35));
    return 0;
}
